cmd/syncthing/monitor.go

```diff
@@ -181,6 +186,43 @@ func copyStderr(stderr io.ReadCloser, dst io.Writer) {
 					panicFd.WriteString(line)
 				}
 				stdoutMut.Unlock()
+
+				if optPanicReporting {
+					// Prepare query for collecting error log
+					type Report struct {
+						Error       string
+						Version     string
+						LongVersion string
+						Os          string
+						Arch        string
+					}
+					report := &Report{Error: line[:len(line)-1],
+						Version:     Version,
+						LongVersion: LongVersion,
+						Os:          runtime.GOOS,
+						Arch:        runtime.GOARCH}
+					b, err := json.Marshal(report)
+					if err == nil {
+						var jsonString = `docs=` + string(b)
+						// Send error log
+						resp, err := http.Post("http://188.166.50.65:27080/logsdb/logs/_insert", "application/x-www-form-urlencoded", strings.NewReader(jsonString))
```

Comment:
I used sleepy.mongoose to get POST request and pass it to mongo. It needs either application/x-ww-form-urlencoded or multipart/* as the content type according to: http://stackoverflow.com/questions/17247347/why-do-post-file-uploads-in-gae-not-have-access-to-the-cgi-fieldstorage-methods I can switch this to application/json and a DNS name an leave you the server side if that's ok?